### PR TITLE
ENH: Add week 5 quiz

### PR DESCRIPTION
**Pull request by dmoracze**

### CODE REVIEW

File: week_5/week_5_quiz.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+**Question 1**
+
+ What were the overarching motivations for the tasks selected for the neurocognitive battery?
+
+ - [ ] the tasks maximize sensitivity to development and other emergent factors across a decade of assessment, starting from the ages of 9-10 
+ - [ ]  the tasks minimize participant burden 
+ - [ ]  the tasks are comparable to those used for other large studies 
+ - [ ]  the tasks cover multiple domains
+ - [ ]  all the above
```
Comment (tsalo) [inline week_5/week_5_quiz.md:19]:
```suggestion
 - [ ] the tasks minimize participant burden 
 - [ ] the tasks are comparable to those used for other large studies 
 - [ ] the tasks cover multiple domains
 - [ ] all the above
```

File: week_5/week_5_quiz.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+**Question 2**
+
+This lecture covered the results of Thompson et al, 2019 whereby principal component analysis was applied to 12 neurocognitive assessments. Based on this study, what were the main principal components underlying the different tasks administered to participants at baseline? Select all that apply.
```
Comment (tsalo) [inline week_5/week_5_quiz.md:23]:
```suggestion
This lecture covered the results of Thompson et al. (2019) wherein principal component analysis was applied to 12 neurocognitive assessments. Based on this study, what were the main principal components underlying the different tasks administered to participants at baseline? Select all that apply.
```

File: week_5/week_5_quiz.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+- [ ] Stanford Mental Arithmetic Response Time Eval
+- [ ] Dimensional Change Card Sort
+
+**Question 4**
+
+Which of these are true of the Cash Choice and Delay Discounting tasks? Select all that apply.
+
+- [ ] A strong association has been reported between temporal delay and substance use 
+- [ ] Both of these tasks are administered at each timepoint
+- [ ] The decision participants make regarding receiving $75 in three days or $115 in three months are developmentally stable
```
Comment (tsalo) [inline week_5/week_5_quiz.md:45]:
```suggestion
- [ ] The decision participants make about whether to receive $75 in three days or $115 in three months is developmentally stable 
```
Mostly "are" to "is", but the other proposed wording change makes it clearer, I think.

File: week_5/week_5_quiz.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+ - CC-0
+ - CC-BY
+ - CC-NC
+ - CC-INFC
+
+***
+For the next questions, you’ll need the following sample dataset repository: https://github.com/ABCD-ReproNim/sample_dataset
+
+To get set up, complete the following steps:
+1. Fork the sample_repository to create your own copy
```
Comment (tsalo) [inline week_5/week_5_quiz.md:95]:
```suggestion
1. Fork the sample repository to create your own copy
```

File: week_5/week_5_quiz.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+- [ ] The object’s URL will never change
+- [ ] DOIs are persistent
+- [ ] DOIs are resolvable
+- [ ] DOIs receive a unique accession number
+
+**Question 7**
+
+The [MyConnectome](https://openneuro.org/datasets/ds000031/versions/00001) dataset is a neuroimaging dataset available on openneuro.
+
+Which of the following is(are) globally unique and persistent identifier(s) that point to this dataset? Select all that apply.
```
Comment (tsalo) [inline week_5/week_5_quiz.md:73]:
```suggestion
Which of the following are globally unique and persistent identifier that point to this dataset? Select all that apply.
```

I think having "select all that apply" is enough to convey that there may be one or more applicable options.

File: week_5/week_5_quiz.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+- [ ] Dimensional Change Card Sort
+
+**Question 4**
+
+Which of these are true of the Cash Choice and Delay Discounting tasks? Select all that apply.
+
+- [ ] A strong association has been reported between temporal delay and substance use 
+- [ ] Both of these tasks are administered at each timepoint
+- [ ] The decision participants make about whether to receive $75 in three days or $115 in three months is developmentally stable
+- [ ] All the above
```
Comment (tsalo) [inline week_5/week_5_quiz.md:46]: Can be dropped if "select all that apply" is used.

File: week_5/week_5_quiz.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+
+**Question 1**
+
+ What were the overarching motivations for the tasks selected for the neurocognitive battery?
+
+ - [ ] the tasks maximize sensitivity to development and other emergent factors across a decade of assessment, starting from the ages of 9-10 
+ - [ ] the tasks minimize participant burden 
+ - [ ] the tasks are comparable to those used for other large studies 
+ - [ ] the tasks cover multiple domains
+ - [ ] all the above
```
Comment (tsalo) [inline week_5/week_5_quiz.md:19]: Other questions capitalize answers.

**Comment by tsalo** [comment]
The content looks good to me. Just a couple of formatting things.

**Comment by dmoracze** [comment]
Thanks for the attention to detail @tsalo

**Review by tsalo — APPROVED** [review]
LGTM.

**Pull request merged**